connectors/google-maps-platform/element-templates/google-maps-platform-connector.json

```diff
@@ -0,0 +1,533 @@
+      "label": "Region code",
+      "description": "See a <a href='https://developers.google.com/maps/documentation/address-validation/coverage/'> supported regions</a>, for find needed region code ",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body.address.regionCode"
+      },
+      "constraints": {
+        "notEmpty": false
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "validateAddress"
+        ]
+      }
+    },
+    {
+      "label": "Locality",
+      "description": "An incorporated city or town political entity",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "body.address.locality"
+      },
+      "constraints": {
+        "notEmpty": false
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "validateAddress"
+        ]
+      }
+    },
+    {
+      "label": "Address",
+      "description": "Unstructured address. Detailes in a <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/google-maps-platform/'> connector documantation</a>",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "addressValue"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "validateAddress"
+        ]
+      }
+    },
+    {
+      "label": "Address",
+      "description": "Complete address. Detailes in a <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/google-maps-platform/'> connector documantation</a>",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "addressValue"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "getPlaceID"
+        ]
+      }
+    },
+    {
+      "label": "Destination",
+      "description": "The place ID value to which you wish to calculate directions",
+      "group": "input",
+      "type": "String",
+      "feel": "optional",
+      "optional": false,
+      "binding": {
+        "type": "zeebe:input",
+        "name": "destinationValue"
+      },
+      "constraints": {
+        "notEmpty": true
+      },
+      "condition": {
+        "property": "operationType",
+        "oneOf": [
+          "directions"
+        ]
+      }
+    },
+    {
+      "label": "Origin",
+      "description": "The place ID value from which you wish to calculate directions.",
```

Comment:
replace with: The place ID value that you want to use as the starting point for calculating distance.